RIE/RIE201.xml

```diff
@@ -113,7 +113,12 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                 </ab>
          </div>
          <div type="bibliography">
-           <listBibl type="secondary"><bibl>
+           <listBibl type="secondary">
+             <bibl xml:id="rie1">
+               <ptr target="bm:RIE1"/>
+               <citedRange unit="page">296</citedRange>
+             </bibl>
+            <bibl>
```

Comment:
I notice that sometimes you give `@xml:id` to your `<bibl>` sometimes not. I would suggest that you need to do that only if you need to refer to one of those, e.g. from a `<div>`